docs/server/installation.rst

```diff
@@ -172,23 +172,12 @@ the database and populate it with initial data. This is done by executing the
   (env) $ pootle migrate
   (env) $ pootle initdb
 
-
-.. _installation#refreshing-stats:
-
-Refreshing stats
-----------------
-
-On first installation you will need to generate the statistics from your
-database. You will need to have an :ref:`RQ worker running 
+You will need to have an :ref:`RQ worker running 
 <installation#running-rqworker>` to complete this.
```

Comment:
This is because some readers that are in a hurry might just copy and paste the commands, run them and then finish reading the instructions and notice that they ran something they actually shouldn't.